### ISSUE TITLE
auto scale by 10,000%

### ISSUE THREAD

**Issue by LegoDwarf120** [Ultimaker/Cura#16324]
### Cura Version

5.4.0

### Operating System

windows 11

### Printer

creality

### Reproduction steps

edited a file on 3d builder
then put the file into cura slicer

### Actual results

it just auto scaled by 10,000%

### Expected results

place normally into slicer and not auto scale

### Add your .zip and screenshots here ⬇️

![2023-07-30](https://github.com/Ultimaker/Cura/assets/140945178/0ebf7dc0-1e78-47ac-be2b-8dee04e8dcd4)

**Comment by flame4ever** [comment]
I have the same problem, but also with STL files.
[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/HjZgK5ZP1kA/0.jpg)](https://youtu.be/HjZgK5ZP1kA)

**Comment by GregValiant** [comment]
Cura doesn't just make up numbers, it gets them from the model file.  If you know the height, or width, or length, you can adjust the scale.

**Comment by nallath** [comment]
This option is there if cura detects _reaaaaaly_ small objects. That's also where the number of 10k comes from, as it tries to figure out what you meant. 

You can disable this in the preferences (i think it's called "automatically scale small objects")

**Comment by flame4ever** [comment]
The Prusa Silcer also copes with this and as can be seen in the video, it first has the object at the right size and then makes it larger, can't this be prevented?

**Comment by flame4ever** [comment]
@fieldOfView You're right, I set it to mm and the behaviour is gone, thank you very much!!!
![image](https://github.com/user-attachments/assets/d1a61f6e-51bf-40eb-ac3c-aa4767573e19)

**Comment by GregValiant** [comment]
@flame4ever if this is resolved can we close this?
